### PR TITLE
Adjust the docs to the new documentation guidelines

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Adjust MD files to the new documentation guidelines
- Improve formatting
- Rephrase to eliminate ambiguity
- Replace the no longer used abbreviation "GCP" with "Google Cloud"
- Fix a link

**Related issue(s)**
 #9917

### CODE REVIEW

File: cmd/image-builder/README.md [hunk 1]
```diff
@@ -186,9 +186,9 @@ Image Builder supports three build backends:
 
 - kaniko
 - BuildKit
-- Azure DevOps pipelines
+- ADO pipelines
 
-kaniko and BuildKit build images locally while the Azure DevOps pipelines backend call ADO API.
+kaniko and BuildKit build images locally while the ADO pipelines backend call ADO API.
```
Comment (NHingerl) [inline cmd/image-builder/README.md:191]:
```suggestion
kaniko and BuildKit build images locally, while the ADO pipelines backend call ADO API.
```
I'd use a comma here, and also I don't understand this sentence: "the ADO pipelines backend call ADO API". Who calls? The pipelines? Or the backend?

File: cmd/image-url-helper/README.md [hunk 2]
```diff
@@ -52,9 +52,9 @@ go run main.go \
     --dry-run=false
 ```
 
-### Exclude images from the check command
+### Exclude Images from the Check Command
 
-To exclude image lines from being checked, create a YAML file that contains a list of files and values of images that you want to exclude from the check. Then, provide a path to this file in the `check` command argument. See the example of such a YAML file:
+To exclude image lines from checking, create a YAML file that contains a list of files and values of images that you want to exclude from the check. Then, provide a path to this file in the `check` command argument. See the example of such a YAML file:
```
Comment (NHingerl) [inline cmd/image-url-helper/README.md:57]: I'm not sure this should be converted to active voice

File: cmd/tools/clusterscollector/README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-This command finds and removes orphaned clusters created by the `kyma-gke-integration` job in a Google Cloud Platform (GCP) project.
+This command finds and removes orphaned clusters created by the `kyma-gke-integration` job in a Google Cloud (GCP) project.
```
Comment (NHingerl) [inline cmd/tools/clusterscollector/README.md:5]: is GCP as an abbreviation still correct?

File: cmd/tools/diskscollector/README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-This command finds and removes orphaned disks created by the `kyma-gke-integration` job in a Google Cloud Platform (GCP) project.
+This command finds and removes orphaned disks created by the `kyma-gke-integration` job in a Google Cloud (GCP) project.
```
Comment (NHingerl) [inline cmd/tools/diskscollector/README.md:5]: again, I'm not sure GCP is still a correct abbreviation

File: cmd/tools/dnscollector/README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-This command finds and removes orphaned IP Addresses and related DNS records created by GKE integration jobs in a Google Cloud Platform (GCP) project.
+This command finds and removes orphaned IP Addresses and related DNS records created by GKE integration jobs in a Google Cloud (GCP) project.
```
Comment (NHingerl) [inline cmd/tools/dnscollector/README.md:5]: again, I'm not sure GCP is still a correct abbreviation

File: cmd/tools/gcrcleaner/README.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
-# GCR cleaner
+# GCR Cleaner
 
 ## Overview
 
-This command finds and removes old GCR images created by Jobs in the Google Cloud Platform (GCP) project.
+This command finds and removes old GCR images created by Jobs in the Google Cloud (GCP) project.
```
Comment (NHingerl) [inline cmd/tools/gcrcleaner/README.md:5]: again, I'm not sure GCP is still a correct abbreviation

File: cmd/tools/ipcleaner/README.md [hunk 7]
```diff
@@ -2,14 +2,14 @@
 
 ## Overview
 
-This command finds and removes orphaned IP addresses created by jobs in the Google Cloud Platform (GCP) project.
+This command finds and removes orphaned IP addresses created by jobs in the Google Cloud (GCP) project.
```
Comment (NHingerl) [inline cmd/tools/ipcleaner/README.md:5]: again, I'm not sure GCP is still a correct abbreviation

**Review by NHingerl — COMMENTED** [review]
Excellent!

I'm not sure GCP is still a correct abbreviation. However, you can choose to ignore that for internal docs.

File: cmd/tools/clusterscollector/README.md (hunk 3 above)
Comment (Sawthis) [reply cmd/tools/clusterscollector/README.md:5]: correct, it's not, good catch
Google Cloud Platform is now called Google Cloud: https://cloud.google.com/blog/topics/developers-practitioners/introducing-new-homepage-google-cloud

GCP could be abbreviation for Google Cloud services.
I think Google Cloud (GCP) it's not correct, it should be rather Google Cloud (GC) project if abbreviation is needed.

File: cmd/tools/diskscollector/README.md (hunk 4 above)
Comment (Sawthis) [reply cmd/tools/diskscollector/README.md:5]: correct, it's not, good catch
Google Cloud Platform is now called Google Cloud: https://cloud.google.com/blog/topics/developers-practitioners/introducing-new-homepage-google-cloud

GCP could be abbreviation for Google Cloud services.
I think Google Cloud (GCP) it's not correct, it should be rather Google Cloud (GC) project if abbreviation is needed.

File: cmd/tools/dnscollector/README.md (hunk 5 above)
Comment (Sawthis) [reply cmd/tools/dnscollector/README.md:5]: correct, it's not, good catch
Google Cloud Platform is now called Google Cloud: https://cloud.google.com/blog/topics/developers-practitioners/introducing-new-homepage-google-cloud

GCP could be abbreviation for Google Cloud services.
I think Google Cloud (GCP) it's not correct, it should be rather Google Cloud (GC) project if abbreviation is needed.

File: cmd/tools/gcrcleaner/README.md (hunk 6 above)
Comment (Sawthis) [reply cmd/tools/gcrcleaner/README.md:5]: correct, it's not, good catch
Google Cloud Platform is now called Google Cloud: https://cloud.google.com/blog/topics/developers-practitioners/introducing-new-homepage-google-cloud

GCP could be abbreviation for Google Cloud services.
I think Google Cloud (GCP) it's not correct, it should be rather Google Cloud (GC) project if abbreviation is needed.

File: cmd/tools/ipcleaner/README.md (hunk 7 above)
Comment (Sawthis) [reply cmd/tools/ipcleaner/README.md:5]: correct, it's not, good catch
Google Cloud Platform is now called Google Cloud: https://cloud.google.com/blog/topics/developers-practitioners/introducing-new-homepage-google-cloud

GCP could be abbreviation for Google Cloud services.
I think Google Cloud (GCP) it's not correct, it should be rather Google Cloud (GC) project if abbreviation is needed.

File: cmd/image-builder/README.md (hunk 1 above)
Comment (dekiel) [reply cmd/image-builder/README.md]: The Image Builder client calls ADO API when using ADO pipelines backend. The call to the ADO API is used to start ADO pipeline, where we execute the real image build process.

Maybe this one would be better?

kaniko and BuildKit build images locally, while for the ADO pipelines backend, Image Builder calls ADO API to start build process.

File: docs/how-to/how-to-create-standard-terraform-config.md [hunk 8]
```diff
@@ -1,26 +1,26 @@
-# Standard Terraform configuration
+# Standard Terraform Configuration
 
-This document describes the standard Terraform configuration that is used in `test-infra` repository. 
+This document describes the standard Terraform configurationuration that is used in the `test-infra` repository.
```
Comment (Sawthis) [inline docs/how-to/how-to-create-standard-terraform-config.md:3]:
```suggestion
This document describes the standard Terraform configuration that is used in the `test-infra` repository. 
```

File: docs/how-to/how-to-create-standard-terraform-config.md [hunk 9]
```diff
@@ -1,26 +1,26 @@
...
+- Reverting the changes if needed and maintaining documentation of the infrastructure along with the code.
 
-## Terraform config structure
+## Terraform Configuration Structure
 
-Our standard structure for Terraform config is based on the [Google Terraform best practices](https://cloud.google.com/docs/terraform/best-practices-for-terraform) and [Hashicorp Creating Terraform Modules](https://developer.hashicorp.com/terraform/language/modules/develop) articles. Thus, we can easily reuse Terraform modules and share them between different projects. We can also easily test the modules on the development environment before applying them to the production environment.
+Our standard structure for Terraform configurationuration is based on the [Google Terraform best practices](https://cloud.google.com/docs/terraform/best-practices-for-terraform) and [Hashicorp Creating Terraform Modules](https://developer.hashicorp.com/terraform/language/modules/develop) articles. Thus, we can easily reuse Terraform modules and share them between different projects. We can also easily test the modules on the development environment before applying them to the production environment.
```
Comment (Sawthis) [inline docs/how-to/how-to-create-standard-terraform-config.md:17]:
```suggestion
Our standard structure for Terraform configuration is based on the [Google Terraform best practices](https://cloud.google.com/docs/terraform/best-practices-for-terraform) and [Hashicorp Creating Terraform Modules](https://developer.hashicorp.com/terraform/language/modules/develop) articles. Thus, we can easily reuse Terraform modules and share them between different projects. We can also easily test the modules on the development environment before applying them to the production environment.
```

**Review by Sawthis — CHANGES_REQUESTED** [review]

**Review by NHingerl — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Review by Sawthis — APPROVED** [review]

**Pull request merged**